packages/shared/src/utils/constants/poi-failure-codes.tsx

```diff
@@ -171,6 +171,10 @@ export const ONFIDO_ERROR_STATUS: TOnfidoErrorStatus = Object.freeze({
         code: 'DataValidationNoDocumentNumbers',
         message: <Localize i18n_default_text='The serial number of your document couldn’t be verified.' />,
     },
+    DuplicatedDocument: {
+        code: 'DuplicatedDocument',
+        message: <Localize i18n_default_text='Your verification documents were already used for another account.' />,
+    },
```

Comment:
is the message content approved by content team

Reply:
It is provided by Laura and confirmed by Wali